.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+Description of the pull request:
+
+Related Issue:
+
+New Feature Submissions:
```

Comment:
I think this doesn't fit well with the project. Instead, add

- Features
- Contribution guide
- Get started
- Signed off 

Also, feel free to use [this](https://embeddedartistry.com/blog/2017/08/04/a-github-pull-request-template-for-your-projects/) if you lack points.